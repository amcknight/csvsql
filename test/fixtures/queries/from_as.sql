SELECT P.first_name, P.age
FROM test/fixtures/tables/people as P
