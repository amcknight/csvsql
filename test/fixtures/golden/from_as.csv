first_name, age
Andrew, 35
Tom, 36
Cayley, 31
July, 0
